uid-pages/layout-bonita/src/LayoutBonitaV4.json

```diff
@@ -388,7 +388,7 @@
                             },
                             "hidden" : {
                               "type" : "expression",
-                              "value" : "!isGuestUser"
```

Comment:
Why the renaming ? SSO can be setup and logout still be visible. With SAML for example you can do a global logout and so you want to keep the link.
I think we should keep the old name